lms/templates/emails_change_successful.html

```diff
@@ -1,18 +0,0 @@
-<%inherit file="main.html" />
```

Comment:
Can you help me understand why you're removing this? I took a quick look and it seems to be unused, is that why?